src/Native/System.Net.Security.Native/CMakeLists.txt

```diff
@@ -30,4 +33,8 @@ target_link_libraries(System.Net.Security.Native
     ${LIBGSS}
 )
 
+include_directories(SYSTEM
+    ${LIBGSS_INCLUDE_DIR}
```

Comment:
Doesn't this append the ${LIBGSS_INCLUDE_DIR} to the current set of included-directories?

Am I right in understanding that the heimdal version of gssapi is not present in the standard include path?

I am worried about the following :  pal_gssapi.cpp includes ```gssapi/gssapi_krb5.h```. This is safe because this is a file that is unique to mit libraries. But ```gssapi_krb5.h``` in turn includes ```gssapi/gssapi.h```.
 
With the above ```include_directories``` isn't it  possible that the file included is from heimdal version but is linked to mit-krb5 version?